package.json

```diff
@@ -23,9 +23,10 @@
     "graphql": "^16.3.0",
     "graphql-request": "^4.0.0",
     "graphql-ws": "^5.5.5",
+    "isomorphic-ws": "^5.0.0",
     "js-sha256": "^0.9.0",
     "querystring-es3": "^0.2.1",
-    "ws": "^8.4.2"
+    "ws": "^8.16.0"
```

Comment:
❔ Why did you downgrade `ws` library? 
Also could it be totally replaced by `isomorphic-ws` library?

Reply:
`ws` is actually upgraded from 4 to 16 😉 It's just unusual to read when version number has 2 digits

`isomorphic-ws` internally uses `ws` for NodeJS and they require to have `ws` package installed as "peer dependency".

Reply:
> ws is actually upgraded from 4 to 16 😉 It's just unusual to read when version number has 2 digits
🙈